composer.json

```diff
@@ -11,6 +11,7 @@
         "ezsystems/platform-ui-assets-bundle": "^2.0.0",
         "ezsystems/repository-forms": "^1.0",
         "ezsystems/ezpublish-kernel": "^6.2",
+        "ezsystems/ez-support-tools": "^0.1-dev",
```

Comment:
I'd really prefer having it as 0.1 than 1.0.0-beta.

What is the problem with 0.x ?